types/src/keyless/circuit_constants.rs

```diff
@@ -56,26 +56,26 @@ pub fn devnet_prepared_vk() -> PreparedVerifyingKey<Bn254> {
 
     let delta_g2 = g2_projective_str_to_affine(
         [
-            "10857046999023057135944570762232829481370756359578518086990519993285655852781",
-            "11559732032986387107991004021392285783925812861821192530917403151452391805634",
+            "6309950375468367434079888575625734658722834850554198467265341412057133512289",
+            "290788916745604303732014379515714703987358626088033030814233237684691015915",
         ],
         [
-            "8495653923123431417604973247489272438418190587263600148770280649306958101930",
-            "4082367875863433681332203403145435568316851327593401208105741076214120093531",
```

Comment:
so in the old vk, `delta_g2 == gamma_g2`, is it supposed to happen?
@mstraka100 @rex1fernando @heliuchuan @alinush

Reply:
I might have forgot to add contributions to the previous dummy setup before we deployed. If both were the g2 generator this would be the case